website/docs/reference/resource-configs/databricks-configs.md

```diff
@@ -437,8 +437,8 @@ Beginning with 1.9, `merge` behavior can be modified with the following addition
 - `skip_not_matched_step`: If set to `true`, the 'not matched' clause will not be included.
 - `matched_condition`: Condition to apply to the `WHEN MATCHED` clause.  You should use the `target_alias` and `source_alias` to write a conditional expression, such as `DBT_INTERNAL_DEST.col1 = hash(DBT_INTERNAL_SOURCE.col2, DBT_INTERNAL_SOURCE.col3)`.  This condition further restricts the matched set of rows.
 - `not_matched_condition`: Condition to apply to the `WHEN NOT MATCHED [BY TARGET]` clause.  This condition further restricts the set of rows in the target that do not match the source that will be inserted into the merged table.
-- `not_matched_by_source_condition`: Condition to apply to the further filter `WHEN NOT MATCHED BY SOURCE` clause.  Only used in conjunction with `not_matched_by_source_action: delete`.
-- `not_matched_by_source_action`: If set to `delete`, a `DELETE` clause is added to the merge statement for `WHEN NOT MATCHED BY SOURCE`.
+- `not_matched_by_source_condition`: Condition to apply to the further filter `WHEN NOT MATCHED BY SOURCE` clause.  Only used in conjunction with `not_matched_by_source_action`.
+- `not_matched_by_source_action`: Action to apply when the condition is met.  For example: `update set t.attr1 = 'deleted', t.tech_change_ts = current_timestamp()`.
```

Comment:
@benc-db This wording is unclear to me. Is this a boolean field or one that takes inputs like delete (and maybe others?)

Reply:
It takes an expression, like the example.